client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-source-url/index.tsx

```diff
@@ -105,8 +105,9 @@ const SiteMigrationSourceUrl: FC< SourceURLProps > = function ( {
 				stepName={ stepName }
 				flowName={ flow }
 				className="import__onboarding-page"
-				hideBack
+				hideBack={ navigation?.goBack === undefined }
```

Comment:
Sorry! After the approval I noticed the `site-migration` always pass the `goBack`. So I think we'll need to change the approach here. Maybe we'll also need to send a `hideBack` prop to avoid a big refactor on the other flows?

Reply:
@renatho Good point! I added an inverted flag here [698fef6](https://github.com/Automattic/wp-calypso/pull/93563/commits/698fef6be02d6c60d12f359996454c3e5e684a65) to keep the react semantic. 

I want to avoid cases like

```
<MigrationSourceUrl 
   hideBack={false} />
```

That will work differently of that could cause more confusion.

```
<MigrationSourceUrl  />
```

Reply:
Just one last detail. I think the default value of the `showBack` should be `false` to keep the original behavior to the components not passing the new prop. Otherwise, it starts showing the "Back" without any behavior in the other flows.

Reply:
Update here [3afe162](https://github.com/Automattic/wp-calypso/pull/93563/commits/3afe1625d0fd7274dbaa6e7c922bbee19a0b0be6)